CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-- ...
+- **[UPDATE]** Add a way to attach specifications to components (with the example of the Button)
```

Comment:
[DOC] ?

Reply:
It's not only an addition of the readme, it's the way we create the stories etc.